### ISSUE TITLE
Work plan (old)

### ISSUE THREAD

**Issue by ramess101** [ramess101/IFPSC_10#3]
@mostafa-razavi 
I just created a Mie_14_Bayesian_Viscosity_200_MCMC directory for butane. I modified the nAlkanes... script so that model is Mie_14... and compound is C4H10. **Can you start those simulations?** It should take the same length of time as your last simulations, i.e. about 6 days.

I am currently rerunning the propane Mie_16 200 MCMC results (it takes about 3 days). The first time I accidentally used the box sizes from ethane, so they were highly compressed liquid simulations. On the bright side, I was able to get meaningful viscosities of around 20 cP using Green-Kubo. Most people would argue you need NEMD for that high of a viscosity, but with replicates you can use EMD.

**Comment by mostafa-razavi** [comment]
@ramess101 
> Can you start those simulations? 

I'm about to run simulations for n-butane Mie_14_Bayesian_Viscosity_200_MCMC. 

>It should take the same length of time as your last simulations, i.e. about 6 days.

I believe it's going to take much less than 6 days because we got rid of the production time, right?

Also, in a previous email you mentioned:

>You might want to modify it using maybe NREPS = 15, nt_eq,prod,vis=1, and iRep from 0 to 12 (this would submit 195 replicates, which is plenty)

I'm going to assign these variables in nAlkanes_BayesianSaturatedViscosity_tabulated:
1- pinoffset=1
2- nt_vis=1
3- nt_eq=1
3- NREPS=16
4- for iRep in $(seq 0 11)

This way I will have (16*12=192) replicates. Does that sound alright?

**Comment by ramess101** [comment]
@mostafa-razavi 

> I'm about to run simulations for n-butane Mie_14_Bayesian_Viscosity_200_MCMC.

Actually, can you kill this job? Why don't we just run the Potoff 16-6 potential for butane first? To make sure we are getting reasonable results. This only requires 30 replicates, so it should only take a day or so.

**Comment by ramess101** [comment]
@mostafa-razavi 

I have updated the directories so that there is now a C4H10/Gromacs/Potoff directory. The nAlkanesBayesian script will modify the cutoff to 1 nm automatically. With NREPS = 16 you will want to change iRep in $(seq 0 1) to have 32 replicates. That should be much faster.

**Comment by ramess101** [comment]
> I believe it's going to take much less than 6 days because we got rid of the production time, right?

Correct. It should take about half as long I think.

> I'm going to assign these variables in nAlkanes_BayesianSaturatedViscosity_tabulated:
1- pinoffset=1
2- nt_vis=1
3- nt_eq=1
3- NREPS=16
4- for iRep in $(seq 0 11)

Notice that I have commented out most of the pinoffset stuff. This was because my machine was not assigning cores properly and my colleagues were not happy with that. If you are the only user on your machine you should be fine to use the -pinoffset option for gmx directly. 

> This way I will have (16*12=192) replicates. Does that sound alright?

Yes that is correct.

**Comment by ramess101** [comment]
@mostafa-razavi 

I have 40 replicate simulations running for C3H8 with the TAMie force field without the production step. I have also included two scripts for nAlkanes and branchedAlkanes that do not require MCMC parameter sets, i.e. they use a single literature set of epsilon/sigma.

**Comment by mostafa-razavi** [comment]
> Actually, can you kill this job? Why don't we just run the Potoff 16-6 potential for butane first? To make sure we are getting reasonable results. This only requires 30 replicates, so it should only take a day or so.

I'm running Potoff n-butane now. I'm also post-processing Mie n-butane 16 results. It takes longer than I thought. It must be ready in a few hours.

**Comment by ramess101** [comment]
> I'm running Potoff n-butane now.

Great. Maybe after that you can run TAMie and TraPPE butane. We will then probably want to compare our TAMie, Potoff, and TraPPE results for propane and butane to see if they look reasonable. Then we will want to move on to octane and see how many molecules we need to get a good viscosity. We want to figure that out before we start submitting all the branched alkane jobs, which currently use 800 molecules each.

> I'm also post-processing Mie n-butane 16 results. It takes longer than I thought. It must be ready in a few hours.

Yes it does take a long time. That is primarily because it is performing 100 bootstrap fits for each density. Fortunately, we only have to run that once.

**Comment by mostafa-razavi** [comment]
@ramess101 
Strangely enough, one can modify other people's comments in Github. I don't know if the other person will get notified or not. If yes, it would be great if we keep a list of simulations in the first comment of this issue and keep updating it. I'm thinking to include simulations that are finished, those that are running and those that needs to be done. This way we will have a better big picture of what needs to be done. What do you think?

**Comment by ramess101** [comment]
@mostafa-razavi 

Wow, that is strange. I never noticed that before. I will create this in a new issue.
